### PR TITLE
feat(colorscheme): add nord

### PR DESCRIPTION
**Pull request by 0xricksanchez**

### CODE REVIEW

File: lua/astrocommunity/colorscheme/nord-nvim/README.md
Comment (owittek) [inline lua/astrocommunity/colorscheme/nord-nvim/README.md:1]: Could you please adapt the customization info for lazy.nvim?

e.g.

```lua
local utils = require("astronvim.utils")
{
  "lukas-reineke/headlines.nvim",
  opts = function(_, opts)
    if not markdown then markdown = {} end
    opts.markdown.headline_highlights = utils.list_insert_unique(opts.markdown.headline_highlights, {
      "Headline1", 
      -- ...
    )}
  -- ...
  end
}
```

Feel free to look at other plugins in this repo to get a feel for this!

File: lua/astrocommunity/colorscheme/nord-nvim/README.md [hunk 1]
```diff
@@ -2,43 +2,52 @@
...
-    bold = true,
-})
-
-require("bufferline").setup({
-    options = {
+local utils = require("astronvim.utils")
+{
+  "akinsho/nvim-bufferline"
+  opts = function(_, opts)
+    if not options then options = {} end
```
Comment (owittek) [inline lua/astrocommunity/colorscheme/nord-nvim/README.md:40]:
```suggestion
    if not opts.options then opts.options = {} end
```

File: lua/astrocommunity/colorscheme/nord-nvim/README.md [hunk 2]
```diff
@@ -2,43 +2,52 @@
...
-        codeblock_highlight = "CodeBlock",
-        dash_highlight = "Dash",
-        quote_highlight = "Quote",
-    },
-})
+local utils = require("astronvim.utils")
+{
+  "lukas-reineke/headlines.nvim",
+  opts = function(_, opts)
+    if not markdown then markdown = {} end
```
Comment (owittek) [inline lua/astrocommunity/colorscheme/nord-nvim/README.md:16]:
```suggestion
    if not opts.markdown then opts.markdown = {} end
```

**Review by owittek — DISMISSED** [review]
I missed this in my suggestion, whoops

**Review by luxus — DISMISSED** [review]
why are you using init and config?

**Comment by 0xricksanchez** [comment]
> why are you using init and config?

Couldn't make it work with using `opts`. The current version at least seems to use the options from the `init` function. It does seem the `nord` theme doesn't have a `setup()` that can take opts

Also I had marked this as @owittek and I were debugging the additional configuration options on Discord (those mentioned in the `README.md`). They don't seem to apply yet and I have no idea why. If you can shed some light on that I'd be glad to make the appropriate changes to get this merged.

File: lua/astrocommunity/colorscheme/nord-nvim/nord.lua [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "shaunsingh/nord.nvim",
+  init = function()
+    vim.g.nord_contrast = true
+    vim.g.nord_borders = false
+    vim.g.nord_disable_background = false
+    vim.g.nord_italic = false
+    vim.g.nord_uniform_diff_background = true
+    vim.g.nord_bold = false
+  end,
+  config = function() require("nord").set() end,
```
[the remark this answers is not in the document]
Comment (Uzaaft) [reply lua/astrocommunity/colorscheme/nord-nvim/nord.lua:11]: This was recommended by me.  See https://discord.com/channels/939594913560031363/939857762043695165/1103574684777324574

**Review by mehalter — APPROVED** [review]
made a small simplification but looks great otherwise!

**Pull request merged**